### PR TITLE
DM Liquibase changesets IBP-6253 IBP-6212

### PR DESCRIPTION
**Pull request by mchabito**

### CODE REVIEW

File: src/main/resources/liquibase/crop_changelog/23_4_0.xml [hunk 1]
```diff
@@ -25,4 +25,464 @@
 		<comment>Create json_props column in phenotype_aud table</comment>
 		<addColumn tableName="phenotype_aud"><column name="json_props" type="CLOB"></column></addColumn>
 	</changeSet>
+
```
Comment (abatac) [inline src/main/resources/liquibase/crop_changelog/23_4_0.xml:28]: Please remove the changes here in 23_4_0.xml.

File: src/main/resources/liquibase/crop_master.xml [hunk 2]
```diff
@@ -104,6 +104,7 @@
 	<include file="crop_changelog/23_4_0.xml" relativeToChangelogFile="true"/>
 	<include file="crop_changelog/24_2_0.xml" relativeToChangelogFile="true"/>
 
+
```
Comment (abatac) [inline src/main/resources/liquibase/crop_master.xml:107]: Include the changelog file here:
```<include file="crop_changelog/24_3_0.xml" relativeToChangelogFile="true"/>```

File: src/main/resources/liquibase/crop_changelog/24_3_0.xml [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+        </sql>
+    </changeSet>	
+	
+    <changeSet id="v24.3.0-16" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Plant Number" AND cv_id = 1010;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm Plot Number </comment>
```
Comment (abatac) [inline src/main/resources/liquibase/crop_changelog/24_3_0.xml:343]: Comment should be "Add cvterm Plant Number"

File: src/main/resources/liquibase/crop_changelog/24_3_0.xml [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+        </sql>
+    </changeSet>	
+	
+	<changeSet id="v24.3.0-14" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Replication Number" AND cv_id = 1010;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm Plot Number </comment>
```
Comment (abatac) [inline src/main/resources/liquibase/crop_changelog/24_3_0.xml:289]: Comment should be "Add cvterm Replication Number"

File: src/main/resources/liquibase/crop_changelog/24_3_0.xml [hunk 5]
```diff
@@ -0,0 +1,466 @@
...
+				LEFT JOIN (
+					SELECT DISTINCT entryid, listid,gid,lrecid  FROM listdata ORDER BY listid, entryid 
+					) AS zz
+				ON z.listid = zz.listid AND z.zentryid  = zz.entryid ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Update listdata with sequence number</comment>
+        <sql dbms="mysql" splitStatements="true">
+		
+				SET @seq := 0;
```
Comment (abatac) [inline src/main/resources/liquibase/crop_changelog/24_3_0.xml:407]: Are the SQL for updating the entry id of listdata and stock performant? Do they only affect the listdata/stock records that are not sequential?

**Review by abatac — CHANGES_REQUESTED** [review]

File: src/main/resources/liquibase/crop_master.xml (hunk 2 above)
Comment (mchabito) [reply src/main/resources/liquibase/crop_master.xml:107]: done! cc @rvtulod

File: src/main/resources/liquibase/crop_changelog/24_3_0.xml (hunk 4 above)
Comment (mchabito) [reply src/main/resources/liquibase/crop_changelog/24_3_0.xml]: comment corrected

File: src/main/resources/liquibase/crop_changelog/24_3_0.xml (hunk 3 above)
Comment (mchabito) [reply src/main/resources/liquibase/crop_changelog/24_3_0.xml]: comment corrected

File: src/main/resources/liquibase/crop_changelog/23_4_0.xml (hunk 1 above)
Comment (mchabito) [reply src/main/resources/liquibase/crop_changelog/23_4_0.xml]: done!

**Pull request closed without merging**